.cicd/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Did we decide to use an bash script for our build CMD? If so, we should go ahead and port this across the other Dockerfiles in a later PR. Just something to note.

Reply:
This was something we discussed in the meeting on Tuesday (see PR description, paragraph 2).    
   
I should have done this in a separate PR, but I was bored waiting for builds so I added commit [`cc14159fe661e77bb425357d3f7b4750251b8b78`](https://github.com/EOSIO/eos-vm/pull/45/commits/cc14159fe661e77bb425357d3f7b4750251b8b78) and commit [`3c849fb3ab037f95eff51d5a053151699ef060c0`](https://github.com/EOSIO/eos-vm/pull/45/commits/3c849fb3ab037f95eff51d5a053151699ef060c0). They are both granular so we can revert them if that's what the group decides.

Reply:
Gotcha. I'll approve and we can discuss further later.